app/celery/scheduled_tasks.py

```diff
@@ -402,3 +406,17 @@ def check_job_status():
             queue=QueueNames.JOBS
         )
         raise JobIncompleteError("Job(s) {} have not completed.".format(job_ids))
+
+
+@notify_celery.task(name='daily-stats-template_usage_by_month')
+@statsd(namespace="tasks")
+def daily_stats_template_usage_my_month():
+    results = dao_fetch_monthly_historical_stats_by_template()
+
+    for result in results:
+        insert_or_update_stats_for_template(
+            result.template_id,
+            result.month.month,
+            result.year.year,
```

Comment:
why do you have to do `result.month.month` here? what does `result.month` return, and can we change the dao fn so we don't need to do this?